websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Cards
+---
+
+ * The actual card content is markdown
+ * Title have no linkable / hover anchor like section headings
+ * Card titles never become entries in the page index nav like section headings
+ * Either an icon or an image path can be set, but not both.
+   * To start with, Images are _not_ supported for free-form in-content cards, only in places that are fully designed and implemented by developers. An Image is sized into the width of the card.
+   * Icons are supported anywhere but only those that are officially provided by the kit. An icon is sized into a fixed 48x48px square.
+ * Cards can occur in two interaction variants, "clickable" (visually elevated) and "flat" (visually an outline only).
+ * Cards must be embedded into a wrapper component that defines the responsive sizes (two aside / three aside) and parameters that must be identical in a given row or grid (clickable card area, the heading size)
+   * trial & error led to a minimum width of ca 250px for narrow three-aside cards and ca 332px for two-aside cards.
+ * A single card must not fill the full width but stay left aligned.
+ * Multiple cards fill the full width and are scaled up to fill even if their content is smaller. Every card has the same width in all rows (resulting in a grid if it's multiple rows)
+ * Cards are sorted in rows, then columns.
+
+# "Cards" Wrapper
+
+ Properties (for authors):
+ * `clickable` (boolean): if set, the whole area of all child cards is a link, the card is elevated and has a hover effect if a link url is set.
+ * `narrow` (boolean): by default, two cards are shown side by side (except mobile viewports). if `narrow` is passed, the minimum card width is reduced so that three cards fit the content width on microsite landing pages.
+ * `smallTitle` (boolean): lets all cards have a smaller title font.
+
+## Minimal form
+
+```jsx
+<Cards>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+```
+### Output
+
+<Cards>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+
+## All options configured
+
+```jsx
+<Cards clickable narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+```
+
+### Output
+
+<Cards clickable narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+# "Card": the actual card
+
+Properties (for authors):
+ * `title` (string): the card title, no markdown allowed.
+ * `href` (string): the link target of the bottom link and, if `clickable` is set, the whole card. `href` has no effect when neither `clickable` or `textLink` are set.
```

Comment:
I think we can get rid of the `clickable` option and just derive the "type" of cards if the `href` property is set.

We need to perform some validations anyway, like if one of the cards in a group of cards has a link, all of them should have a link.

Reply:
No, this does not suffice, if a card has only the textLink that link must use the href location, too but that does not mean that the card is supposed to be elevated and completely clickable. 

For example, if you have a row of cards that are not elevated / clickable and some of them get a text link (but others don't have a link) you need to switch the whole row to be non-clickable although the ones with a text link at the bottom still need the href.   

separate href targets for the whole card and the bottom text links are not allowed.

Reply:
Sorry if I'm still asking questions but I haven't fully wrapped my head around this yet.

So for me to understand, let me try to summarize it again.
- Individual cards can only have one link, which is either assigned to the entire card OR to the bottom link.
- Individual cards don't need to have links.
- The cards wrapper determines the "type" of the group of cards, either "raised" or "flat", no matter if the single cards have links or not.
- If the group of cards should be "flat", only cards with a link AND a "textLink" should get the link rendered at the bottom. Cards that have a link but NOT a "textLink" should not be clickable.
- If the group of cards should be "raised", ALL cards with a link should be clickable (the entire card) even if there is a "textLink" at the bottom.
  - **in this scenario, should all cards be required to have link?**

Hope I got this right this time.